tests/Unit/Domain/Creators/Test_Sphere.cpp

```diff
@@ -439,10 +466,13 @@ void test_sphere_factory_time_dependent() {
       "Sphere:\n"
       "  InnerRadius: 1\n"
       "  OuterRadius: 3\n"
-      "  InnerCubeSphericity: 0.0\n"
+      "  Interior:\n"
+      "    FillWithSphericity: 0.0\n"
```

Comment:
Maybe add an option construction test with an excised interior now that the shell tests will be removed?

Reply:
I prepended a commit to prepare testing time dependencies, and added two commits to test option-creation.